### PR TITLE
PD-1136: Adds border to project select component

### PR DESCRIPTION
**Pull request by brookescarlett**
Screen Shot 2021-02-22 at 4 49 25 PM

## ✍️ Proposed changes

🎟 _Jira ticket:_ [PD-1136](https://jira.mongodb.org/browse/PD-1136)

cc: @nellem23

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

### CODE REVIEW

File: packages/mongo-nav/src/mongo-select/OrgSelect.tsx [hunk 1]
```diff
@@ -57,10 +57,8 @@ import {
 const orgButtonStyle = css`
   justify-content: space-between;
   border-radius: 5px 0 0 5px;
-  border: 1px solid ${uiColors.gray.light2};
```
Comment (DesignerDave) [inline packages/mongo-nav/src/mongo-select/OrgSelect.tsx:60]: Am I missing something, or did we only remove a border, not add one? 🤔

**Review by DesignerDave — CHANGES_REQUESTED** [review]

File: packages/mongo-nav/src/mongo-select/OrgSelect.tsx (hunk 1 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/mongo-select/OrgSelect.tsx:60]: this is a dupe from here: https://github.com/mongodb/leafygreen-ui/blob/main/packages/mongo-nav/src/mongo-select/styles.ts#L45, and the line that I removed from the project select file was simply overwriting this one.

Comment (brookescarlett) [reply packages/mongo-nav/src/mongo-select/OrgSelect.tsx:60]: if you run locally, you should see both borders as in the screenshot attached

**Comment by nellem23** [comment]
@brookescarlett @DesignerDave 

So I think we need to step back and talk about the desired end result:

For Ops Manager we'd like to have a border on the Project Dropdown. This is because the element is pulled out into the side nav and right now is floating in space and looks weird.

For everything else (Cloud, Atlas, Realm, Charts, Cloud Manager) we want to keep the Project Nav as is and *not* include a border on the Project Dropdown.

**Comment by nellem23** [comment]
OpsManager@2x
Atlas@2x

**Comment by brookescarlett** [comment]
@nellem23 if that's the case then we currently support passing a className to the component itself, so where you want to add a border that should just work

**Comment by nellem23** [comment]
@brookescarlett if I understand correctly then, it is an implementation detail for the engineering team?

**Comment by brookescarlett** [comment]
@nellem23 yep

**Pull request closed without merging**